.gitignore

```diff
@@ -1,5 +1,4 @@
 node_modules
 .idea
-dist
```

Comment:
Lets put this back in the gitignore so that we don't commit its contents by accident.